package.json

```diff
@@ -21,9 +21,9 @@
     "lint": "npm run lint-package-json && npm run eslint && lerna run python-lint && npm run lint-md",
     "lint-md": "markdownlint docs/**/*.md docs/*.md tf-modules/**/*.md tf-modules/*.md",
     "lint-package-json": "npmPkgJsonLint .",
-    "test": "lerna run --ignore @cumulus/cumulus-integration-tests --concurrency 1 test",
+    "test": "lerna run --ignore @cumulus/cumulus-integration-tests --concurrency 1 test > testoutput.txt",
```

Comment:
I have this on both but really it's only test:coverage that's run in CI.

Reply:
Is this a needed change then?  What problem is it solving?

Reply:
It's probably YAGNI. I left it in case there was something somewhere running just `test` and not `test:coverage` but all of our main problem here is Bamboo and that runs test:coverage in unit-tests.sh. Removing it from 'test' because you're right that it doesn't solve anything 😄 
https://github.com/nasa/cumulus/pull/3577/commits/5e05d4bf30c26f679df460344ee1047a857f0d47

As an aside, if further investigation is required we could pipe individual package's `test:coverage` outputs instead of the parent package.json (which runs everything). That might be something to look at if we end up not finding useful info in the logs generated here.